other/verify_image.yaml

```diff
@@ -16,7 +16,7 @@ metadata:
       its signature against the provided public key. This policy serves as an illustration for
       how to configure a similar rule and will require replacing with your image(s) and keys.
 spec:
-  validationFailureAction: enforce
+  validationFailureAction: audit
```

Comment:
This one should remain in `enforce` mode because it's pointless to have it in `audit` mode.

Reply:
It seemed to me that other `background: false` validate policies (e.g. block-pod-exec-by-pod-label.yaml) are also useless in `audit` mode. I don't understand why only this `verify_image.yaml` and the next `block_updates_deletes.yaml` should be in `enforce` mode.

Reply:
The samples that block operations are not really valuable in background mode, so setting those to `false` makes sense. And same thing with these being in `audit` mode. If the whole point of the policy is to block updates and deletes, putting in audit does nothing. That's the exception. With the verify image policy, same sort of deal except that policy is really only an illustration, and no user is actually going to use that policy as-is without modification.

Reply:
Does the verifyImages rule allow background scan? I couldn't find any mention of it in the documentation. If background scan is possible, I thought it would be better to specify `background: true`.

Reply:
Kyverno v1.4.2 allows one to create this policy with `background` in either mode. @JimBugwadia what is supported here officially? I'm guessing `background: true` doesn't actually do anything. Can you confirm? If that ends up being the case, shouldn't we prohibit such policies from being created/updated?

Reply:
I have changed the `validationFailureAction` back to `enforce`, and the `background` is `false` as in the original.